sparkhpc/sparkjob.py

```diff
@@ -579,7 +579,7 @@ def start_cluster(memory, cores_per_executor=1, timeout=30, spark_home=None):
     if not os.path.exists(os.path.join(spark_home,'logs')):
         os.makedirs(os.path.join(spark_home,'logs'))
 
-    master_log = '{spark_home}/logs/spark_master.out'.format(spark_home=spark_home)
+    master_log = '/home/%s/.sparkhpc/spark_master.out' % os.environ['USER']
```

Comment:
we don't want to hardcode `/home/<username>` because that might not be the location of the home directory on all systems. Please use `os.path.expanduser('~')` instead. 

Another improvement would be to be able to specify an alternate location for the master log to prevent filling up the user's home directory via an environment variable.